internal/plugin_config_test.go

```diff
@@ -653,6 +653,103 @@ policies:
 	assertEqual(t, err.Error(), expected)
 }
 
+func TestConfigInvalidEvalInterval(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	createConfigMap(t, tmpDir, "configmap.yaml")
+
+	tests := []struct {
+		// Individual values can't be used for compliant/noncompliant since an empty string means
+		// to not inherit from the policy defaults.
+		defaultEvalInterval  string
+		policyEvalInterval   string
+		manifestEvalInterval string
+		expectedMsg          string
+	}{
+		{
+			`{"compliant": "not a duration"}`,
+			"",
+			"",
+			`the policy policy-app has an invalid policy.evaluationInterval.compliant value: time: invalid duration ` +
+				`"not a duration"`,
+		},
```

Comment:
I think at least one of these tests should be an invalid duration like `10d5h1m` or `1w2d` that someone might try, just in case the `time` package behavior changes.